### PR TITLE
JSCOE-83 React 18 Features

### PR DESCRIPTION
**Pull request by JasonOuyang80**

### CODE REVIEW

File: react-18-features/src/api/fetchData.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/* eslint-disable consistent-return */
```
Comment (michaeltintiuc) [inline react-18-features/src/api/fetchData.js:1]: Removed commented out code and probably best to not have eslint overrides

File: react-18-features/src/api/fetchGraphQLData.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      Authorization: `bearer ${GITHUB_AUTH_TOKEN}`,
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      query: text,
+      variables,
+    }),
+  })
+
+  // Get the response as JSON
```
Comment (michaeltintiuc) [inline react-18-features/src/api/fetchGraphQLData.js:19]: The eslint rules makes sense, best to return a promise that the caller can await on instead

**Review by michaeltintiuc — CHANGES_REQUESTED** [review]
Overall looks good, but better remove the eslint overrides as really the ones added are best practices

File: react-18-features/src/api/fetchGraphQLData.js (hunk 2 above)
Comment (JasonOuyang80) [reply react-18-features/src/api/fetchGraphQLData.js:17]: Thanks Michael!

**Review by michaeltintiuc — APPROVED** [review]

**Pull request merged**